frappe/email/doctype/email_account/email_account.py

```diff
@@ -289,6 +314,16 @@ def insert_communication(self, msg):
 			# dont count emails sent by the system get those
 			raise SentEmailInInbox
 
+		name = frappe.db.get_value("Communication", { "unique_id": unique_id })
```

Comment:
this currently will set uid's accross accounts and would b very expensive db wise
can be removed if new sync stratergy is used